easybuild/easyconfigs/m/MView/MView-1.67-foss-2022a-Perl-5.34.1.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = "Tarball"
+
+name = 'MView'
+version = "1.67"
+
+homepage = 'http://sourceforge.net/projects/bio-mview/'
+description = """ MView reformats the results of a sequence database search or a 
+ multiple alignment, optionally adding HTML markup."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
```

Comment:
@buzh You're using a full toolchain here, but I doubt that MView required MPI or BLAS/LAPACK, so this should be lowered to a compiler-only toolchain like `GCC/11.3.0` (which is compatible with `foss/2022a`)

Reply:
Sounds right, thanks. I've made an update now.